README.md

```diff
@@ -2,54 +2,76 @@
 
 # Stager
 
-Stager enables rapid Flutter development and encourages good architectural practices by allowing developers quickly launch and develop isolated portions of an app.
+Stager is a Flutter development tool that allows you to run small portions of your app as individual Flutter apps. This lets you:
 
-Stager can accelerate your development workflow for widgets that:
+- Focus your development on a single widget or flow – no more clicking through multiple screens or setting external feature flags to reach the page you're working on.
+- Ensure your UI works in *all* cases, including:
+  - Light and Dark mode
+  - Small or large text sizes
+  - Different viewport sizes
+  - Different device types
+  - Loading, empty, error, and normal states
+- Show all of this to your designers to make sure your app is pixel-perfect.
 
-- Have multiple states (empty, error, loading, etc.) that would otherwise require code changes to trigger.
-- Are cumbersome to navigate to.
-- Are hidden behind a feature flag.
-- Behave differently based on external state (e.g., the type of currently logged-in user).
+## Demo
+
+![example app demo](https://user-images.githubusercontent.com/581764/219502623-bfe44091-a582-460e-b97d-e786bc614a8c.gif)
+
+The example included in this repo demonstrates how Stager can be used in the context of a simple Twitter-like app that displays a feed of posts and includes detail pages for posts and users.
+
+Stager uses Scenes (see the [Concepts](#concepts) section below) that you define to generate small Flutter apps. To run the Stager apps included in the example, start by moving to the `example` directory and fetching the app's dependencies:
+
+```bash
+cd example
+flutter pub get
+```
+
+You can then run the indivdual Stager apps with the following commands:
+
+**Posts List**
+```bash
+flutter run -t lib/pages/posts_list/posts_list_page_scenes.stager_app.g.dart
```

Comment:
does pub get generate these files or you need to run something first?

Reply:
They were initially generated with build runner, but are checked in to the repo. I generally prefer examples/demos that don't require a bunch of setup to run. I'll add a note about this.